cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke1-custom.po.ts

```diff
@@ -37,4 +38,28 @@ export default class ClusterManagerCreateRke1CustomPagePo extends ClusterManager
   nodeCommand(): EmberAccordionPo {
     return new EmberAccordionPo('cluster-driver__role');
   }
+
+  clusterAgentAccordion() {
+    return new EmberAccordionPo('cluster-driver__cluster-agent');
+  }
+
+  fleetAgentAccordion() {
+    return new EmberAccordionPo('cluster-driver__fleet-agent');
+  }
+
+  clusterAgentConfiguration() {
+    return new EmberAgentConfigurationPo('[data-testid="form-cluster-agent"]');
+  }
+
+  fleetAgentConfiguration() {
+    return new EmberAgentConfigurationPo('[data-testid="form-fleet-agent"]');
+  }
+
+  next() {
```

Comment:
next and done should be available via ClusterManagerCreateRKE1PagePo

Reply:
removed